RELEASING.md

```diff
@@ -18,74 +18,58 @@ Also check if you can upgrade any dependencies:
 make update-dependency-versions
 ```
 
-## Make the release ##
+## Decide what the next version should be ##
+
+This depends on what's changed (see `CHANGELOG.md`):
 
-In order to make releases secrets are required. Members of the core team can
-install keybase and join the `cucumberbdd` team to access these secrets.
+* Bump `MAJOR` if:
+  * There are `Changed` or `Removed` entries
+  * A cucumber library dependency upgrade was major
+* Bump `MINOR` if:
+  * There are `Added` entries
+* Bump `PATCH` if:
+  * There are `Fixed` or `Deprecated` entries
```

Comment:
Deprecations can only happen on a minor version.

> https://semver.org/
> 7. Minor version Y (x.Y.z | x > 0) MUST be incremented if new, backwards compatible functionality is introduced to the public API. It MUST be incremented if any public API functionality is marked as deprecated. It MAY be incremented if substantial new functionality or improvements are introduced within the private code. It MAY include patch level changes. Patch version MUST be reset to 0 when minor version is incremented.

Reply:
Yep, updated